src/averageAges.js

```diff
@@ -14,12 +14,28 @@
  *
  * @return {number}
  */
+
+function callbackSum(summa, n) {
```

Comment:
Consider renaming the function 'callbackSum' to a more descriptive name like 'sumAgeDifference'.

Reply:
Consider renaming `summa` to `sum`, this is a correct word